.goreleaser.yaml

```diff
@@ -67,3 +67,30 @@ scoop:
   homepage: https://pomdtr.github.io/sunbeam
   description: Generate powerful UIs from simple scripts written in any language.
   license: MIT
+aurs:
+  - name: sunbeam-bin
+    homepage: https://pomdtr.github.io/sunbeam
+    description: Generate powerful UIs from simple scripts written in any language.
+    maintainers:
+      - 'Achille Lacoin  <achille@deta.sh>'
+    license: MIT
+    # IMPORTANT: the key must not be password-protected.
+    private_key: '{{ .Env.AUR_KEY }}'
+    git_url: 'ssh://aur@aur.archlinux.org/sunbeam-bin.git'
+    conflicts:
+      - sunbeam
+      - sunbeam-bin
```

Comment:
why does sunbeam-bin conflicts with sunbeam-bin ?

Reply:
Good catch, no idea what I was thinking :P Rebased to fix this.